nanome/_internal/_structure/_complex.py

```diff
@@ -105,4 +110,17 @@ def _convert_to_conformers(self, force_conformers = None):
 
     def _convert_to_frames(self):
         result = _helpers._conformer_helper.convert_to_frames(self)
-        return result
\ No newline at end of file
+        return result
+
+    def _on_complex_updated(self, new_complex):
+        self.__copy_received_complex(new_complex)
+        self._complex_updated_callback()
```

Comment:
Can we pass `new_complex` to the callback here? That way a callback function can be reused for multiple complexes and it will know which complex was updated.